src/string.cr

```diff
@@ -491,7 +491,7 @@ class String
     invalid : Bool
 
   private macro gen_to_(method, max_positive = nil, max_negative = nil)
-    info = to_u64_info(base, whitespace, underscore, prefix, strict)
+    info = to_u64_info(base, whitespace, underscore, prefix, strict, unsigned: {{max_negative == nil}})
```

Comment:
I did that because at all the other arguments you can also see what the name of this argument is (because the variables are named after the argument names) and so I wanted to adapt this one to the others by also clearly saying what the name of this argument is. It's also faster to understand so you don't have to scroll down to the `to_u64_info` def first.